test/functional/ui/screen.lua

```diff
@@ -385,6 +385,51 @@ function Screen:_current_screen()
   return table.concat(rv, '\n')
 end
 
+function Screen:snapshot_util(attrs)
+  -- util to generate screen test
+  self:wait(function()
+  end, 250)
```

Comment:
If you meant to wait 250 milliseconds, the right way would be to return an error in the callback and catch it with `pcall`. For example:

```lua
pcall(function() self:wait(function() return 'err stub' end) end) -- should only return after 250 ms
```